src/reanimated2/threads.ts

```diff
@@ -15,6 +15,7 @@ export function setupMicrotasks() {
   'worklet';
 
   let microtasksQueue: Array<() => void> = [];
+  let callMicrotasksRunning = false;
```

Comment:
"running" doesn't not feel right here, can we rename to sth along the lines of `isExecutingMicrotasksQueue`